src/local-mode.ts

```diff
@@ -1,12 +1,59 @@
 import * as Bluebird from 'bluebird';
 import * as Docker from 'dockerode';
 import * as _ from 'lodash';
-
```

Comment:
I generally leave these newlines as a separator for internal and external dependencies, at a glance. It's not a hard and fast rule, but it does lay out the imports better when there's quite a few.

Reply:
It's changed because I auto-organize imports with my editor. Will check if it can allow configuration like this.

Reply:
There is no such option :(
Do you know if we can configure prettify to do this?

Reply:
I don't believe so. I'd prefer our imports didn't reorganise with every commit in a different file though.

Reply:
Yep, completely agree - I removed this change.